internal/pkg/v2/controller/errorconcept/default.go

```diff
@@ -0,0 +1,110 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package errorconcept
+
+import (
+	"net/http"
+)
+
+const (
+	METHOD_NOT_ALLOWED = "isA should not be invoked, this is to only be used as a default error concept"
```

Comment:
typo isA

Reply:
This const is invoked by all the `isA` functions.